internal/adapters/storage/postgres/scheduler/scheduler_storage_pg.go

```diff
@@ -397,6 +397,7 @@ func (s schedulerStorage) RunWithTransaction(ctx context.Context, transactionFun
 	transactionID := ports.TransactionID(uuid.New().String())
 	s.transactionsMap[transactionID] = transaction
 
+	//nolint use any(nil)
```

Comment:
Instead of disable the lint, remove the `any` casting in line 402

Reply:
the value that `err` receive is of type `any`, so it forces the cast

Reply:
I inputted the specific rule for this too

Reply:
I removed because its just warm in my local lint